target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -102,7 +40,7 @@ ramips_setup_interfaces()
 		ucidef_set_interface_lan "lan"
 		;;
 	*)
-		ucidef_set_interfaces_lan_wan "eth0.1" "eth0.2"
+		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" "wan"
```

Comment:
I never checked whether devices actually fell into the old default case here. While I agree that the new default makes sense, have you checked whether all devices in mt7621 were accounted for explicitly here before? (This is _not_ a request, I'm just curious whether you did it anyway when moving config...)

Reply:
I just checked. There are 78 MT7621 boards and no one used this default case

Reply:
`Acked-by: Adrian Schmutzler <freifunk@adrianschmutzler.de>` for this commit.